lib/dalli/protocol/meta/request_formatter.rb

```diff
@@ -28,10 +28,10 @@ def self.meta_get(key:, value: true, return_cas: false, ttl: nil, base64: false,
 
         def self.meta_set(key:, value:, bitflags: nil, cas: nil, ttl: nil, mode: :set, base64: false, quiet: false)
           cmd = "ms #{key} #{value.bytesize}"
-          cmd << ' c' unless %i[append prepend].include?(mode)
+          cmd << ' c' if !quiet && !%i[append prepend].include?(mode)
```

Comment:
we have found that this shouldn't be sent most of the time, but definitely not in quiet mode